projects/plugins/jetpack/modules/subscriptions/views.php

```diff
@@ -274,6 +274,7 @@ static function render_widget_subscription_form( $args, $instance, $subscribe_em
 		$subscribers_total            = self::fetch_subscriber_count();
 		$subscribe_placeholder        = isset( $instance['subscribe_placeholder'] ) ? stripslashes( $instance['subscribe_placeholder'] ) : '';
 		$submit_button_classes        = isset( $instance['submit_button_classes'] ) ? $instance['submit_button_classes'] : '';
```

Comment:
I think this would be tidier by removing the new line and adding the handling here (this way avoids a possible awkward space in the final markup).

<img width="410" alt="Screen Shot 2021-09-08 at 12 42 28 PM" src="https://user-images.githubusercontent.com/349751/132575120-9de6ef50-32af-426b-958c-6ac46984dae2.png">

```suggestion
		$submit_button_classes        = isset( $instance['submit_button_classes'] ) ? 'wp-block-button__link ' . $instance['submit_button_classes'] : 'wp-block-button__link';
```